src/web/aws-cloudwatch/CloudWatch.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Row } from 'react-bootstrap';
+
+// import UserNotification from 'util/UserNotification';
+import Wizard from 'components/common/Wizard';
+
+import CloudWatchAuthorize from './StepAuthorize'
```

Comment:
When importing React components like this, I feel confused because I expect the file name to match the name of the (default) import. Would it be ok for you renaming either the import or the filename?